src/odoc/bin/main.ml

```diff
@@ -137,18 +128,28 @@ end = struct
     in
     Arg.(value & opt_all string default & info ~docv:"MODULE" ~doc ["open"])
 
+  let children =
+    let doc = "Specify the odoc file as a child. Can be used multiple times. Only applies to mld files" in
+    let default = [] in
+    Arg.(value & opt_all string default & info ~docv:"CHILD" ~doc ["child"])
```

Comment:
Cmdliner has a `list` option type that takes several values separated by a `,`, would it be more readable/easier to type on the CLI ? (`value & list string & ...`)

Reply:
I thought it would be better to be consistent with the other CLI arguments.

Reply:
Make sense. Maybe we should have a single letter option then ? I don't know any tool that have "long" options that are expected to be repeated.
The options will be seen in build systems logs and maybe wrote by hand sometimes, so readability matters.

Reply:
Ah, good point! I had forgotten that.